pytest_inmanta/parameters.py

```diff
@@ -63,6 +63,18 @@
     group=param_group,
 )
 
+inm_no_strict_deps_check = BooleanTestParameter(
+    argument="--no-strict-deps-check",
+    environment_variable="INMANTA_NO_STRICT_DEPS_CHECKS",
+    usage=(
+        "Tell pytest-inmanta to run without using the deps check after module installation."
+        "When using the dependency check, an error is raised if there are conflicting requirements"
+        "when disabling the check, a warning is shown but no error is raised"
```

Comment:
```suggestion
        "Tell pytest-inmanta to run without using the deps check after module installation."
        " When using the dependency check, an error is raised if there are conflicting requirements"
        " when disabling the check, a warning is shown but no error is raised"
```
String concatenation does not add spaces.